appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -484,8 +484,10 @@ private YaVersion() {
   // - POLYGON_COMPONENT_VERSION was incremented to 2
   // - RECTANGLE_COMPONENT_VERSION was incremented to 2
   // - TEXTBOX_COMPONENT_VERSION was incremented to 6
+  // For YOUNG_ANDROID_VERSION 195:
+  // - PEDOMETER_COMPONENT_VERSION was incremented to 2
 
-  public static final int YOUNG_ANDROID_VERSION = 194;
+  public static final int YOUNG_ANDROID_VERSION = 195;
```

Comment:
Revert this change as we will update this only once everything else is ready to go.